sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_aggregates_base/view.sql

```diff
@@ -15,17 +82,17 @@ SELECT
     TIMESTAMP_TRUNC(`timestamp`, HOUR),
     INTERVAL((DIV(EXTRACT(MINUTE FROM `timestamp`), 5) + 1) * 5) MINUTE
   ) AS window_end,
-  COUNTIF(event_method = 'enroll') AS enroll_count,
-  COUNTIF(event_method = 'unenroll') AS unenroll_count,
-  COUNTIF(event_method = 'graduate') AS graduate_count,
-  COUNTIF(event_method = 'update') AS update_count,
-  COUNTIF(event_method = 'enrollFailed') AS enroll_failed_count,
-  COUNTIF(event_method = 'unenrollFailed') AS unenroll_failed_count,
-  COUNTIF(event_method = 'updateFailed') AS update_failed_count
+  COUNTIF(event_category = 'enroll' OR event_category = 'enrollment') AS enroll_count,
+  COUNTIF(event_category = 'unenroll' OR event_category = 'unenrollment') AS unenroll_count,
+  COUNTIF(event_category = 'graduate') AS graduate_count,
+  COUNTIF(event_category = 'update') AS update_count,
+  COUNTIF(event_category = 'enrollFailed') AS enroll_failed_count,
+  COUNTIF(event_category = 'unenrollFailed') AS unenroll_failed_count,
+  COUNTIF(event_category = 'updateFailed') AS update_failed_count,
+  COUNTIF(event_category = 'disqualification') AS disqualification_count,
+  COUNTIF(event_category = 'exposure') AS exposure_count
```

Comment:
this is `expose` on desktop; let's capture those, too. https://searchfox.org/mozilla-central/source/toolkit/components/telemetry/Events.yaml#724-728